docs/legacy/v2-products/v2-v3.md

```diff
@@ -46,7 +46,7 @@ Creating Products and Variants on V3:
 
 Variants can be included with a GET request to lower the number of API calls being made using `?include=variants`.
 
-V3 includes an endpoint for creating a [Catalog Tree](/api-reference/catalog/catalog-api).
+V3 includes an endpoint for retrieving a [Catalog Tree](/api-reference/store-management/catalog/category/getcategorytree).
```

Comment:
1. this redirect 404s. fix is to point at whatever Mark says.
2. post MSF launch, there can now be more than one tree and trees belong to the catalog directly, not to categories. point to the MSF api guide.  
3. log/find out whether the new category tree endpoints apply only to MSF enabled stores. MSF was supposed to not introduce breaking changes for single SF stores. unsure whether the old endpoint is still documented, or the slug just changed & some of these endpoints are the same.
4. find or create tickets to recheck broken redirects and set human-readable links to all the new MSF endpoints.
5. starting point as follows:
```suggestion
V3 includes endpoints for working with catalog trees. Stores that have multi-storefront enabled can have more than one tree. See the ... section of the [Multi-Storefront API Guide](/api-docs/multi-storefront/api-guide#...). Storesthat do not have multi-storefront enabled ...
```

Reply:
@markcmurphy Are the Category Tree endpoints [here ](https://developer.bigcommerce.com/api-reference/b3A6NDgxNDA1MTI-get-a-category-tree)used for both MSF and non-MSF?